src/index.js

```diff
@@ -16,10 +16,22 @@ export default function App () {
           </Route>
         </Routes>
       </BrowserRouter>
-   <Services />
-    <Footer />
-  </div>
-
+      <Services />
+      <ServiceArea />
+        <div className='flex flex-col p-16 bg-green-700'>
+        <h1 className='text-center text-white'>
+          Icons from &quot;Lawn Care&quot; collection, created by Berkah Icon,
+          from{' '}
+          <a
+            className='underline'
+            href='https://thenounproject.com/browse/collection-icon/lawn-care-163681/?p=1'
+          >
+            Noun Project
+          </a>{' '}
+          CC BY 3.0
+        </h1>
+      </div>
+    </div>
```

Comment:
It looks like this is overwriting the footer from the main branch. The copyright changes have been merged into main so i think you should be able to merge main into your branch and not need to make any changes to this? i'm not completely sure what's going on but please view your local version and/or deploy preview and compare it to what's deployed live already at [yorkshirewo](https://yorkshirewoodsgroundcrew.com/) to make sure you're not introducing a regression into the site.